src/UI/Component/Layout/Page/Factory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace ILIAS\UI\Component\Layout\Page;
+
+use ILIAS\UI\Component\MainControls;
+
+/**
+ * This is what a factory for pages looks like.
+ */
+interface Factory
+{
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *    The Standard Page is the regular view upon ILIAS.
+	 *
+	 *   composition: >
+	 *      The main parts of a Page are the Metabar hosting the logo and tools,
```

Comment:
I think it would be worth discussing, if the logo should be part of the metabar or not. The alternative would be to pass the logo directly to the Page Layout. This would allow to locate the logo at a place more distant from the metabar (of course the page layout could "extract" the logo from the metabar but this feels somehow wrong).

Reply:
Sounds plausible to me, what do you think @nhaagen?

Reply:
All fine with me.